js/apps/system/_admin/aardvark/APP/frontend/js/views/collectionsView.js

```diff
@@ -650,7 +650,7 @@
                 false,
                 [
                   {
-                    rule: Joi.string().allow('').optional().regex(/^[1-9]*$/),
+                    rule: Joi.string().allow('').optional().regex(/^[1-9][0-9]*$/),
```

Comment:
Nit:
Just mentioning here that there is also a maximum value for writeConcern and for replicationFactor.
The current hard-coded maximum value is 10, although the effective maximum value for replicationFactor is the minimum of the number of available DB servers and 10.
The effective maximum value for writeConcern is the effective maximum value of replicationFactor.
The current validation will allow entering much higher values, which will then be rejected by the server.

I don't think it is a good idea to hard-code all these validations in the frontend anyway, as any change in the server will leave the frontend validations untouched and silently get out of sync.
I don't suggest doing any change here, but probably we should not put too much effort into the frontend validations now until we have a better approach of keeping the frontend and server side validation in sync.